### PR TITLE
Js Wrapper Service Worker

### LINKED ISSUE #24: Js Wrapper Service Worker
**Is your feature request related to a problem? Please describe.**
Adding Js Wrapper Service Worker.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
_List Changes Introduced by this PR_
1. Added the Js wrapper service worker

## Purpose
This will auto cache the data

Closes #24

### CODE REVIEW

File: pokeapp/pokeapi-js-wrapper-sw.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const imgRe = /https:\/\/raw\.githubusercontent\.com\/PokeAPI\/sprites\/[\/-\w\d]+\/[\d\w-]+\.(?:png|svg|gif)/
+const version = 1
```
Comment (Lap343) [inline pokeapp/pokeapi-js-wrapper-sw.js:2]: interesting regex

**Review by Lap343 — APPROVED** [review]
Code looks good

File: pokeapp/pokeapi-js-wrapper-sw.js (hunk 1 above)
Comment (krferrell) [reply pokeapp/pokeapi-js-wrapper-sw.js:2]: I thought so too. This is straight from the Github of the pokewrapper.

Comment (kmausolf) [reply pokeapp/pokeapi-js-wrapper-sw.js:2]: Yeah, it just caches all the images that are hosted on that website.

**Pull request merged**